src/components/Modals/EditProjectModal/EditProjectModal.tsx

```diff
@@ -28,45 +28,35 @@ export default class EditProjectModal extends React.PureComponent<Props, State>
     }
   }
 
-  handleOnClose = () => {
-    this.props.onClose('EditProjectModal')
-  }
-
   handleSubmit = () => {
     const { title, description, selectedGround } = this.state
-    const { currentProject, grounds } = this.props
+    const { currentProject, grounds, onSave, onSetGround, onClose } = this.props
     const ground = selectedGround ? grounds[selectedGround] : null
     if (currentProject) {
-      this.props.onSave(currentProject.id, { title, description })
-      this.props.onSetGround(currentProject, ground)
+      onSave(currentProject.id, { title, description })
+      onSetGround(currentProject, ground)
     }
-    this.handleOnClose()
+    onClose()
   }
 
   handleTitleChange = (event: React.FormEvent<HTMLInputElement>) => {
-    this.setState({
-      title: event.currentTarget.value
-    })
+    this.setState({ title: event.currentTarget.value })
   }
 
   handleDescriptionChange = (event: React.FormEvent<HTMLInputElement>) => {
-    this.setState({
-      description: event.currentTarget.value
-    })
+    this.setState({ description: event.currentTarget.value })
   }
 
   handlePickGround = (id: string | null) => {
-    this.setState({
-      selectedGround: id
-    })
+    this.setState({ selectedGround: id })
   }
 
   render() {
-    const { modal, grounds } = this.props
+    const { grounds, name, onClose } = this.props
     const { title, description, selectedGround } = this.state
 
     return (
-      <Modal open={modal.open} className="ProjectDetailsModal" size="small" onClose={this.handleOnClose}>
+      <Modal name={name}>
```

Comment:
why remove `className="ProjectDetailsModal"`?

Reply:
`name` is passed down as `className` by the Modal container